cypress/integration/recordings.spec.js

```diff
@@ -12,7 +12,7 @@ context('Recordings', function () {
       cy.contains('a', 'wâpamêw')
         .click()
       cy.url()
-        .should('contain', '/lemma/')
+        .should('contain', '/word/')
```

Comment:
one day I'll figure out how to reverse urls in Cypress

Reply:
lol, fair enough!